Script for manually running perf tests against DC/OS

wavesoft opened the pull request:
This commit introduces the `manual_run_dcos.sh` that can be used to run
the performance tests against an already deployed DC/OS cluster.

_Part of JIRA: DCOS-20295_

jeschkies commented on tests/performance/manual_run_dcos.sh:
```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```
Out of curiosity, did you run [shellcheck](https://github.com/koalaman/shellcheck) against this script?

wavesoft replied:
Nope, but did now and applied some interesting fixes 😄

jeschkies commented on tests/performance/config/perf-driver/environments/target-cluster-custom.yml:
```diff
@@ -0,0 +1,36 @@
+
+# Global test configuration
+# ===========================
+config:
```
Is this already interpreted by the per driver? Where is its implementation?

wavesoft replied:
Yes, this is part of the [standard driver configuration](http://dcos-performance-test-driver.readthedocs.io/en/latest/config/Readme.html). 

The driver itself is supposed to be available in the user's environment before running the manual script.

jeschkies commented:
@wavesoft, what is the state of this diff?

wavesoft commented:
Key @jeschkies , for some reason your comments slipped through my e-mail filters 😄 I addressed your comments so it should be ready to merge.

I have used this script to run the tests on my local infrastructure, so the actual PR is not needed. I just thought it would be nice to have it on our toolbox.
